code/modules/mob/living/living.dm

```diff
@@ -482,25 +482,28 @@
 /mob/living/proc/disable_lights(armor = TRUE, guns = TRUE, flares = TRUE, misc = TRUE, sparks = FALSE, silent = FALSE)
 	return FALSE
 
-/mob/living/update_tint()
-	tinttotal = get_total_tint()
+
+/mob/living/proc/adjust_tinttotal(tint_amount)
+	tinttotal += tint_amount
```

Comment:
Current method works by just adding and diminishing vision impairment instead of the old system of using the biggest one. If you want to change this to the old behavior let me know and I'll help you through it.

Reply:
adding and subtracting makes sense. keep it as is. (gotta punish those punks who want to wear sunglasses and heavy helmets at night.)

Reply:
![JCDenton2](https://user-images.githubusercontent.com/42041276/68534129-7a54ec00-030f-11ea-8e73-db45eed5d9c4.jpg)